target.go

```diff
@@ -68,4 +73,30 @@ func (t *Target) processBuildArgs(lex *shell.Lex, args map[string]string, allowA
 
 func (t *Target) fillDefaults() {
 	t.Dependencies.fillDefaults()
+	t.Image.fillDefaults()
+}
+
+func (i *ImageConfig) fillDefaults() {
+	if i == nil {
+		return
+	}
+
+	i.Post.fillDefaults()
+}
+
+func (p *PostInstall) fillDefaults() {
+	if p == nil {
+		return
+	}
+
+	for oldpath := range p.Symlinks {
+		cfg := p.Symlinks[oldpath]
+		if cfg.Path != "" {
+			cfg.Paths = append(cfg.Paths, cfg.Path)
+			cfg.Path = ""
+		}
+
+		sort.Strings(cfg.Paths)
```

Comment:
I don't think this should be sorted since this should be left to the user.

Reply:
done.